GameServer.cpp

```diff
@@ -254,6 +254,23 @@ GameServer::GameServer(){
         }
     };
 
+    handlers["discard"] = [&](WebSocket* conn, json data){
```

Comment:
I guess we should try to have as many comments as this function has in most of the less obvious parts of the codebase.